tests/spreadsheet_test.py

```diff
@@ -36,10 +36,12 @@ def test_get_worksheet(self):
         sheet1 = self.spreadsheet.get_worksheet(0)
         self.assertTrue(isinstance(sheet1, gspread.Worksheet))
 
-    @pytest.mark.vcr()
+    @pytest.mark.vcr(allow_playback_repeats=True)
     def test_get_worksheet_by_id(self):
-        sheet1 = self.spreadsheet.get_worksheet_by_id(0)
-        self.assertTrue(isinstance(sheet1, gspread.Worksheet))
+        sheet1_by_int = self.spreadsheet.get_worksheet_by_id(0)
+        sheet1_by_str = self.spreadsheet.get_worksheet_by_id(0)
+        self.assertTrue(isinstance(sheet1_by_int, gspread.Worksheet))
+        self.assertTrue(isinstance(sheet1_by_str, gspread.Worksheet))
```

Comment:
it may also be nice to set up a test for del_worksheet_by_id, to see that it works with both `str` and `int` id. This should be a quick addition.

Reply:
I personally prefer tests to be seperate, but that's fine too :)